website/registrations/models.py

```diff
@@ -60,7 +60,11 @@ class Entry(models.Model):
     )
 
     length = models.CharField(
-        verbose_name=_("membership length"), choices=MEMBERSHIP_LENGTHS, max_length=20,
+        verbose_name=_("membership length"),
+        choices=MEMBERSHIP_LENGTHS,
+        max_length=20,
+        help_text="A discount of €7,50 will be applied if you upgrade your year membership to a"
```

Comment:
This help text is global, it should be in de RenewalForm so that it does not show in the registration form.